src/ray/gcs/gcs_server/gcs_actor_distribution.cc

```diff
@@ -104,8 +107,12 @@ GcsBasedActorScheduler::AllocateNewActorWorkerAssignment(
 
 scheduling::NodeID GcsBasedActorScheduler::AllocateResources(
     const ResourceRequest &required_resources) {
+  // TODO(Shanly): Use BundleSpread scheduling policy for the timebeing, it should be
+  // replaced with Spread policy once the shceduling interfaces are unified inside
+  // `ISchedulingPolicy`.
+  auto scheduling_options = SchedulingOptions::BundleSpread();
```

Comment:
For an actor scheduling, should we use `SchedulingOptions::Hybrid()` by default?

Reply:
> For an actor scheduling, should we use `SchedulingOptions::Hybrid()` by default?
It's ok to use `SchedulingOptions::Hybrid()`, but in  order to keep the logic unchanged, I will not modify this in this PR.